uaclient/apt.py

```diff
@@ -340,6 +341,27 @@ def run_apt_update_command(
     return out
 
 
+def update_sources_list(sources_list: str):
+    retry_sleeps = APT_RETRIES.copy()
+    try:
+        while True:
+            try:
+                apt.cache.Cache().update(sources_list=sources_list)
```

Comment:
I'd suggest getting the cache from `uaclient.apt.get_apt_cache` for consistency and safety - that'll help on the tests I am working on right now

Reply:
Done.